js/index.js

```diff
@@ -18,3 +19,77 @@ for (let i = 0; i < skills.length; i++) {
 
   skillsList.appendChild(skill);
 }
+
+// Lesson 4.3
+// Hides messages section if there are no messages
+displayMessages = document.getElementById("messages");
+displayMessages.style.display = "none";
+const lis = displayMessages.querySelector("ul").children;
+
+// appends message and userInfo to message list
+const messageForm = document.getElementsByName("leave_message")[0];
+messageForm.addEventListener("submit", (event) => {
+  event.preventDefault();
+
+  //grabs user info and message and puts it into newMessage variable
+  const usersName = event.target.usersName.value;
+  const usersEmail = event.target.usersEmail.value;
+  let usersMessage = event.target.usersMessage.value;
+  const newMessage = document.createElement("li");
+  newMessage.innerHTML = `<a href='mailto:${usersEmail}'>${usersName} </a><br /><span>${usersMessage}</span><br />`;
+
+  //creates a remove button
+  const removeButton = document.createElement("button");
+  removeButton.innerText = "remove";
+  removeButton.type = "button";
+
+  //creates an edit button
+  const editButton = document.createElement("button");
+  editButton.innerText = "edit";
+  editButton.type = "button";
+
+  //delegates event listener to section with ID of #messages
+  const messageSection = document.getElementById("messages");
```

Comment:
Don't need a second variable pointing to the message section.

Reply:
I cleaned up these variable as you were right I had created two different variables for the same element.